src/mono/wasm/host/RunConfiguration.cs

```diff
@@ -41,7 +41,7 @@ public RunConfiguration(string runtimeConfigPath, string? hostArg)
 
         HostProperties = rconfig.RuntimeOptions.WasmHostProperties;
         if (HostProperties == null)
-            throw new Exception($"Failed to deserialize {runtimeConfigPath} - config");
+            throw new Exception($"Could not find any {nameof(RuntimeOptions.WasmHostProperties)} in {runtimeConfigPath}");
```

Comment:
Why this is mandatory ?

Reply:
What would be OK default value ?

Reply:
We generate the runtimeconfig.json as part to the build. And use this to figure out what js/wasi engine to run the app with.